org/teams.yaml

```diff
@@ -87,6 +87,7 @@ teams:
             description: Maintainers of the English documentation.
             members:
               - adammil2000
+              - davidhauck
```

Comment:
Also add yourself to WG - Docs Maintainers/Infra (line 105)

Reply:
Done